modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,120 @@
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-describe[secrets-mgr:tls-context:mule:describe]| Shows details of a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-list[secrets-mgr:tls-context:mule:list]| Lists all Mule TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-modify[secrets-mgr:tls-context:mule:modify]| Modifies a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-replace[secrets-mgr:tls-context:mule:replace]| Replaces an existing Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-create[secrets-mgr:tls-context:flex-gateway:create]| Creates a new Flex Gateway TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-describe[secrets-mgr:tls-context:flex-gateway:describe]| Shows details of a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-list[secrets-mgr:tls-context:flex-gateway:list]| Lists all Flex Gateway TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-modify[secrets-mgr:tls-context:flex-gateway:modify]| Modifies a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-replace[secrets-mgr:tls-context:flex-gatway:replace]| Replaces an existing Flex Gateway TLS context secret
+
+
+|===
+
+// end::summary[]
+
+// tag::commands[]
+
+[[secret-group-create]]
+== secrets-mgr:secret-group:create
+
+----
+> secrets-mgr:secret-group:create [flags] 
+----
+
+This Command creates a new secret group with the name specified in `--name`.
+
+Prompt the `--downloadable` flag if you want this group to be downloadable by members of your organization.
+
+This command also accepts the default `--help` flag.
+
+
+[[secret-group-delete]]
+== secrets-mgr:secret-group:delete
+
+----
+> secrets-mgr:secret-group:delete [flags] <id>
+----
+This command deletes the secret group specified in <id> +
+
+[WARNING]
+This command does not prompt twice before deleting. If you send a delete instruction, it does not ask for confirmation.
+
+This command accepts only the default flag `--help`.
+
+[[secret-group-describe]]
+== secrets-mgr:secret-group:describe
+
+----
+> secrets-mgr:secret-group:describe [flags] <id>
+----
+
+Returns the details of a secret group specified in <id>. 
+
+This command accepts only the default flag `--help`.
+
+[[secret-group-list]]
+== secrets-mgr:secret-group:list
+
+----
+> secrets-mgr:secret-group:list [flags]
+----
+
+Displays all your secret groups. It returns the name of the secret group, and the ID.
+
+This command accepts only the default flag `--help`.
+
+[[secrets-group-modify]]
+== secrets-mgr:secret-group:modify
+
+----
+> secrets-mgr:secret-group:modify  [flags] <id> 
+----
+
+This command modifies a secret group you pass in `<id>`. +
```

Comment:
id must be passed in using the '--id' flag.   

None of the commands in secrets manager plugin accepts an argument. All values are passed through the flag.
Please see the "secret group" tab in the following spreadsheet for flags that are applicable for each command.

https://docs.google.com/spreadsheets/d/18fz2mC2HdtkJRcrSL-u2AbsFTLp5KSHZiQ25tJ066YU/edit#gid=1533941102